test/e2e/common/security_context.go

```diff
@@ -130,21 +129,24 @@ var _ = framework.KubeDescribe("Security Context", func() {
 		}
 
 		/*
-		  Release : v1.12
-		  Testname: Security Context: readOnlyRootFilesystem=true.
-		  Description: when a container has configured readOnlyRootFilesystem to true, write operations are not allowed.
-		  This test is marked LinuxOnly since Windows does not support creating containers with read-only access.
+			Release : v1.15
+			Testname: Security Context, readOnlyRootFilesystem=true.
+			Description: Container is configured to run with readOnlyRootFilesystem to true which will force containers to run with a read only root file system.
+			Write Operation MUST NOT be allowed and Pod MUST be in Failed state.
+			At this moment we are not considering this test for Conformance due to use of SecurityContext.
+			[LinuxOnly]: This test is marked as LinuxOnly since Windows does not support creating containers with read-only access.
 		*/
 		It("should run the container with readonly rootfs when readOnlyRootFilesystem=true [LinuxOnly] [NodeConformance]", func() {
 			createAndWaitUserPod(true)
 		})
 
 		/*
-		  Release : v1.12
```

Comment:
Just a question for myself, is the release here indicating when the test was last updated or when the test was introduced?

Reply:
**Release: v1.15** :- Indicates Kubernetes release version when the e2e is added to the Conformance suite.
Ref: https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/conformance-tests.md#conformance-test-comment-metadata

Reply:
But this test is not promoted to conformance test at this time, why do we need to update the `Release`?

Reply:
Just to represent that "Conformance Doc" is updated during v1.15 release cycle. Here Release version has no importance as being "Conformance Doc" instead appears just as a comment about what this e2e verifies.